cockatrice/src/window_sets.cpp

```diff
@@ -53,16 +55,33 @@ WndSets::WndSets(QWidget *parent) : QMainWindow(parent)
     connect(aBottom, SIGNAL(triggered()), this, SLOT(actBottom()));
     setsEditToolBar->addAction(aBottom);
 
+    // search field
+    searchField = new QLineEdit;
+    searchField->setObjectName("searchEdit");
+    searchField->setPlaceholderText(tr("Search by set name, code, or type"));
+    searchField->addAction(QPixmap("theme:icons/search"), QLineEdit::LeadingPosition);
+    searchField->setClearButtonEnabled(true);
+    setFocusProxy(searchField);
+
+    resetSortButton = new QPushButton(tr("Restore order"));
```

Comment:
What about `Back to default view`.
That could stress that it's only the displayed view which is going to change, it won't reset your personalized order or restore (=overwrite) changes you made.

It could also show the "mode" you are currently in and highlight that.
`Mode: Picture load ordering` for the disabled button, and `Adjust card art display` for the clickable button.

But that might not yet be thought-out enough... 😄